delete_me.py

```diff
@@ -0,0 +1,3 @@
+good_var_name = 1
+very_bod_vor_name = 1
```

Comment:
format: typos:
Possible typos in `very_bod_vor_name`, suggestions:
Candidate #1 with a confidence 34%
```suggestion
ver_body_var_name = 1
```
Candidate #2 with a confidence 33%
```suggestion
ver_body_for_name = 1
```
Candidate #3 with a confidence 32%
```suggestion
every_body_var_name = 1
```